src/main/java/denis_grimaliuc/actions/BaseActions.java

```diff
@@ -96,6 +98,21 @@ public void shouldHaveTextEndsWith(WebElement element, String text) {
         wait.until(driver -> element.getText().endsWith(text));
     }
 
+    public String getCountsOfWebElements(WebElement element) {
```

Comment:
Move it in your POM class

Reply:
@dgrimaliuc Done, moved it to my PetSection. Please check.